From 3aa62e6b6f441e2265f9c81eb09a37da70b13ac1 Mon Sep 17 00:00:00 2001
From: ArenM <aren@peacevolution.org>
Date: Fri, 5 May 2023 17:47:22 -0400
Subject: [PATCH sxmo-utils 5/6] Don't kill tailtextlog when changing
 recipients

The lsof / grep call is broken and causes the script to crash when
adding or removing a recipient.

Lsof also doesn't have a stable output format, and the format is
different between the busybox and gnu versions, so we shouldn't be
trying to parse the output of it.
---
 scripts/modem/sxmo_modemtext.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/scripts/modem/sxmo_modemtext.sh b/scripts/modem/sxmo_modemtext.sh
index 976519b..e07bb9f 100755
--- a/scripts/modem/sxmo_modemtext.sh
+++ b/scripts/modem/sxmo_modemtext.sh
@@ -127,7 +127,6 @@ sendtextmenu() {
 						mv "$SXMO_LOGDIR/$OLDNUMBER/draft.attachments.txt" \
 							"$SXMO_LOGDIR/$NUMBER/draft.attachments.txt"
 					fi
-					kill "$(lsof | grep "/$OLDNUMBER/sms.txt" | cut -f1)"
 					[ -e "$SXMO_LOGDIR/$NUMBER/sms.txt" ] || touch "$SXMO_LOGDIR/$NUMBER/sms.txt"
 					sxmo_hook_tailtextlog.sh "$NUMBER" &
 				fi
@@ -161,7 +160,6 @@ sendtextmenu() {
 						mv "$SXMO_LOGDIR/$OLDNUMBER/draft.attachments.txt" \
 						"$SXMO_LOGDIR/$NUMBER/draft.attachments.txt"
 					fi
-					kill "$(lsof | grep "/$OLDNUMBER/sms.txt" | cut -f1)"
 					[ -e "$SXMO_LOGDIR/$NUMBER/sms.txt" ] || touch "$SXMO_LOGDIR/$NUMBER/sms.txt"
 					sxmo_hook_tailtextlog.sh "$NUMBER" &
 				fi
-- 
2.40.0

